docs/guides/quickstart.md

```diff
@@ -15,10 +15,10 @@ This page provides the bare-minimum overview on how to start building with WebCo
 We have prepared a [step-by-step walkthrough](../tutorial/1-building-your-first-webcontainers-app) for you!
 :::
 
-**The WebContainer API starter is the fastest way to explore the API**. Open it in StackBlitz editor:
+**The WebContainer API starter is the fastest way to explore the API**. Open it in StackBlitz editor:  
```

Comment:
A "non-desirable" I found was `block` links which often made hit areas in seemingly blank page space, accidentally clickable.
Fixing this made these images sit inline and mess up the corresponding line height.

The "double space" `  ` in markdown is like writing `<br/>` in html, so we're just forcing the images to sit on a new line.

Reply:
Ohhhh interesting. Thank you for the context.